Sources/OpenAPI/Generated/Entities/Platform.swift

```diff
@@ -9,4 +9,5 @@ public enum Platform: String, Codable, CaseIterable {
 	case ios = "IOS"
 	case macOs = "MAC_OS"
 	case tvOs = "TV_OS"
+    case visionOs = "VISION_OS" // VisionOS manually added since version 3.0 of Apple's OpenAPI spec fails to include it. Reported in FB13208526 27/9/2023.
```

Comment:
I'm unsure why GitHub shows the added line with a different indentation, but in Xcode it looks correct 🤷🏼